### ISSUE TITLE
Can I edit a xlsx file when I have opened this file using EXCEL ?

### ISSUE THREAD

**Issue by lmyhit** [dbzhang800/QtXlsxWriter#173]
你好 ，我想要实现在线的修改，而修改时需要EXCEL文件在电脑中处于打开的状态，我尝试了一下，发现无法在线修改，请问可以通过其他的设置实现这个功能么？

**Comment by VSRonin** [comment]
No, this library will recreate from scratch the file and replace the old one. There's also no way to interact with the "shared workbook" excel functionality at the moment
